.mailmap

```diff
@@ -0,0 +1,2 @@
+Jörn Zaefferer <joern.zaefferer@gmail.com>
+Richard D. Worth <rdworth@gmail.com>
```

Comment:
Usually this is something you configure in your text editor. See that red empty sign above? It means your file ends like this: "Richard D. Worth <rdworth@gmail.com>EOF" instead of "Richard D. Worth <rdworth@gmail.com>\nEOF".